lua/lspconfig/configs/lsp_ai.lua

```diff
@@ -0,0 +1,27 @@
+return {
+  default_config = {
+    cmd = { 'lsp-ai' },
+    filetypes = {},
+    root_dir = nil,
+    single_file_support = true,
+    init_options = {
+      memory = {
+        file_store = vim.empty_dict(),
+      },
+      models = vim.empty_dict(),
+    },
+  },
+  docs = {
```

Comment:
Getting an issue downstream with this being a table instead of just a string, do we need to support the `desc` sometimes being a list of strings ?

Reply:
pr welcome

Reply:
Was just double checking whether it was supposed to only be strings or you do expect tables for this, if it can be changed I can do a quick PR.

Reply:
Yeah that looks like a mistake on my part thanks for the PR :+1: